### ISSUE TITLE
Add unit tests for every model

### ISSUE THREAD

**Issue by mibaumgartner** [delira-dev/vision_torch#23]

**Comment by justusschock** [comment]
But how? We would have to skip most of them

**Comment by mibaumgartner** [comment]
I think regardless of the CI/CD memory issues there should be unit tests so we can run them locally and assure that the models at least have a functioning forward pass. 
Furthermore, I'm not sure how large the memory consumption is when we use the minimal input image size. Right now we are using 3x224x224 even though something like 3x32x32 or 3x64x64 would be sufficient.

**Comment by justusschock** [comment]
Yeah, but the some models may not have adaptive pooling, which causes a fixed input size. But I agree that there should be tests. But ideally we should be able to run many model tests in ci/cd
